src/assets/scripts/main.js

```diff
@@ -1 +1,11 @@
 import 'ramda';
+var a = "1";
+
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Error - Too many blank lines at the end of file. Max of 0 allowed. (no-multiple-empty-lines)